Makefile

```diff
@@ -6,19 +6,36 @@ ENDPOINT_URL ?= http://localhost:8000
 AWS_ACCESS_KEY_ID = local
 AWS_SECRET_ACCESS_KEY = local
 
-tables:
+configure:
+	aws configure set aws_access_key_id ${AWS_ACCESS_KEY_ID}
+	aws configure set aws_secret_access_key ${AWS_SECRET_ACCESS_KEY}
+cache:
 	aws --endpoint-url=${ENDPOINT_URL} --region=${REGION} \
-		dynamodb create-table --table-name=${CACHE_TABLE_NAME} \
-		--attribute-definitions=AttributeName=id,AttributeType=S \
-		--key-schema=AttributeName=id,KeyType=HASH \
-		--billing-mode=PAY_PER_REQUEST && \
+	dynamodb create-table --table-name ${CACHE_TABLE_NAME} \
+	--attribute-definitions \
+		AttributeName=PK,AttributeType=S \
+		AttributeName=SK,AttributeType=S \
+	--key-schema \
+		AttributeName=PK,KeyType=HASH \
```

Comment:
@timkelty are we still using the SK here? I thought we chatted about removing that?

Reply:
This is just for test coverage. We're not actively using a compound key anywhere, but this tests that you can configure it that way.